### PR TITLE
add names and versions

### PR DESCRIPTION
**Pull request by maxim-belkin**
Simply adding name and version information
[Edit]
... and a bunch of `apt-get clean` commands

### CODE REVIEW

File: scratch/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,8 @@
-FROM linuxbrew/linuxbrew as base
+FROM linuxbrew/linuxbrew AS base
 LABEL maintainer="Shaun Jackman <sjackman@gmail.com>"
+LABEL name "Linuxbrew/scratch"
+LABEL version "1.0"
+
```
Comment (sjackman) [inline scratch/Dockerfile]: Please remove this blank line.

File: fedora/Dockerfile [hunk 2]
```diff
@@ -1,7 +1,10 @@
 FROM fedora:26
 LABEL maintainer="Shaun Jackman <sjackman@gmail.com>"
+LABEL name "Linuxbrew/fedora26"
```
Comment (sjackman) [inline fedora/Dockerfile]: Should this one be named `Linuxbrew/fedora` for consistency with the directory name?

**Review by sjackman — CHANGES_REQUESTED** [review]
Thanks, Maxim.

File: fedora/Dockerfile (hunk 2 above)
Comment (maxim-belkin) [reply fedora/Dockerfile]: ooops

**Review by sjackman — APPROVED** [review]

**Pull request merged**